src/main/java/org/cirdles/topsoil/app/TopsoilMainWindow.java

```diff
@@ -68,6 +68,8 @@
     Menu datasetsMenu;
     @FXML
     TabPane dataTableTabPane;
+    @FXML
+    Button saveDataTableButton;
```

Comment:
@emilycoleman, to better understand how I'm interacting with the save button, notice that I've added the two lines above and read [the FXML documentation on controllers](https://docs.oracle.com/javase/8/javafx/api/javafx/fxml/doc-files/introduction_to_fxml.html#controllers). The key is that the variable name here, "saveDataTableButton", is the same as the value of the save button's `fx:id` attribute in [TopsoilMainWindow.fxml](https://github.com/johnzeringue/Topsoil/blob/e2f318c11012d645970d90746509ae021394cb3f/src/main/resources/org/cirdles/topsoil/app/TopsoilMainWindow.fxml#L37).